src/issue/screens/new-issue.screen.js

```diff
@@ -90,17 +92,19 @@ class NewIssue extends Component {
       ).then(issue => {
         navigation.navigate('Issue', {
           issue,
+          headerLeft: false,
```

Comment:
Yes I was referring the tab navigator.

I tested it again today and it works like a charm :
<img width=300 alt="ezgif-4-9d4873f833" src="https://user-images.githubusercontent.com/9287184/29363197-560fb166-828f-11e7-9bbe-512f2b1d6eb2.gif">

😃